cli/labels.js

```diff
@@ -34,8 +34,8 @@ const getLabels = (issueOrIssues) => {
     let labels = [...conf.ghLabels];
     labels.push('snyk');
     if (conf.severityLabel) {
-        const issues = Array.isArray(issueOrIssues) ? issueOrIssues : [];
-        const severities = uniq(issues.map((x) => `severity:${x.severity}`));
+        const issues = Array.isArray(issueOrIssues) ? issueOrIssues : [issueOrIssues];
```

Comment:
Note to reviewers:

I don't understand how this ever worked without this change even in the old API?!?